### PR TITLE
API Updates

### PR DESCRIPTION
**Pull request by richardm-stripe**
## Summary
Contains corrections to #792 which was merged but not released.

We are not releasing `GET /v1/customers/:id/funding_instructions` yet and are releasing only the `POST`, and are generating it as a "custom method" instead of the nested resource method.

### CODE REVIEW

File: stripe/api_resources/customer.py [hunk 1]
```diff
@@ -43,6 +44,12 @@ class Customer(
 ):
     OBJECT_NAME = "customer"
 
+    def create_funding_instructions(self, idempotency_key=None, **params):
+        url = self.instance_url() + "/funding_instructions"
+        headers = util.populate_headers(idempotency_key)
+        self.refresh_from(self.request("post", url, params, headers))
```
Comment (dcr-stripe) [inline stripe/api_resources/customer.py:50]: This shouldn't refresh_from and instead return the funding instructions

**Review by dcr-stripe — CHANGES_REQUESTED** [review]

File: stripe/api_resources/customer.py (hunk 1 above)
Comment (richardm-stripe) [reply stripe/api_resources/customer.py]: Fixed!

File: stripe/object_classes.py [hunk 2]
```diff
@@ -39,7 +39,6 @@
     api_resources.File.OBJECT_NAME: api_resources.File,
     api_resources.File.OBJECT_NAME_ALT: api_resources.File,
     api_resources.FileLink.OBJECT_NAME: api_resources.FileLink,
-    api_resources.FundingInstructions.OBJECT_NAME: api_resources.FundingInstructions,
```
[the remark this answers is not in the document]
Comment (richardm-stripe) [reply stripe/object_classes.py]: Fixed!

**Review by kamil-stripe — APPROVED** [review]

File: stripe/api_resources/terminal/reader.py [hunk 3]
```diff
@@ -53,7 +53,6 @@ class TestHelpers(APIResourceTestHelpers):
         def present_payment_method(self, idempotency_key=None, **params):
             url = self.instance_url() + "/present_payment_method"
             headers = util.populate_headers(idempotency_key)
-            self.resource.refresh_from(
-                self.resource.request("post", url, params, headers)
-            )
-            return self.resource
+            resp = self.resource.request("post", url, params, headers)
```
Comment (pakrym-stripe) [inline stripe/api_resources/terminal/reader.py:56]: This might be a breaking change, we are not refreshing the original resource anymore..

**Pull request merged**